.github/workflows/publish_on_request.yml

```diff
@@ -0,0 +1,105 @@
+name: Publish on request
+
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  build:
+    if: ${{ github.event.issue.pull_request && github.event.comment.body == '/publish'}}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+
+      - uses: actions/checkout@v3
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
+
+      - uses: actions/checkout@v3
+        if: success()
+        with:
+          ref: ${{ steps.comment-branch.outputs.head_ref }}
```

Comment:
This step is triggered two times, is that mandatory ?

Reply:
nop it is an error